app/js/questionNav.js

```diff
@@ -97,6 +100,11 @@ function fillNav() {
     })
 }
 
+// function updateFlagStatus() {
+//     document.querySelector('#flag-checkbox').checked = flaggedQuestions[current]
+// }
```

Comment:
This should also be rid of if you are no longer using it

Reply:
Got rid of them in >> be5a5bf